### PR TITLE
[FEAT] Season Items boosts

### PR DESCRIPTION
**Pull request by Rofrtd**
# Description

This PR implements the boosts for the new items

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/game/expansion/lib/boosts.ts [hunk 1]
```diff
@@ -114,6 +123,13 @@ export const getFoodExpBoost = (
     boostedExp = boostedExp.mul(1.05);
   }
 
+  if (
+    food.name in COOKABLE_CAKES &&
```
Comment (brian-man) [inline src/features/game/expansion/lib/boosts.ts:127]: Now that `COOKABLE_CAKES` exists, can Chef Apron be updated to support Orange Cake?

File: src/features/game/events/landExpansion/buyDecoration.test.ts [hunk 2]
```diff
@@ -269,32 +269,6 @@ describe("buyDecoration", () => {
     });
   });
 
-  it("throws an error if max limit reached", () => {
```
Comment (brian-man) [inline src/features/game/events/landExpansion/buyDecoration.test.ts:272]: In case you're not aware:
#2763

Comment (spencerdezartsmith) [reply src/features/game/events/landExpansion/buyDecoration.test.ts:272]: Fixed.

**Review by spencerdezartsmith — APPROVED** [review]
LGTM 🌽

**Pull request merged**